frontend/talentsearch/src/js/components/browse/PoolCard.tsx

```diff
@@ -0,0 +1,178 @@
+import { Link } from "@common/components";
+import { getPoolStream } from "@common/constants/localizedConstants";
+import React from "react";
+import { useIntl } from "react-intl";
+import Chip, { Chips } from "@common/components/Chip";
+import {
+  getLocale,
+  getLocalizedName,
+  localizeSalaryRange,
+} from "@common/helpers/localize";
+import { formattedDateMonthDayYear } from "@common/helpers/dateUtils";
+import { PoolAdvertisement } from "../../api/generated";
+import { useDirectIntakeRoutes } from "../../directIntakeRoutes";
+
+export interface CardProps {
+  pool: PoolAdvertisement;
+}
+
+const PoolCard = ({ pool }: CardProps & React.HTMLProps<HTMLDivElement>) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useDirectIntakeRoutes();
+
+  const poolClassificationString = (): string | null => {
+    if (pool.classifications && pool.classifications[0]) {
+      const classification = pool.classifications[0];
+      return `${classification.group}-0${classification.level}`;
+    }
+    return null;
+  };
+
+  const poolClassificationSalary = (): string | undefined => {
+    if (pool.classifications && pool.classifications[0]) {
+      const classification = pool.classifications[0];
+      return localizeSalaryRange(
+        classification.minSalary,
+        classification.maxSalary,
+        locale,
+      );
+    }
+    return undefined;
+  };
+
+  const poolEssentialSkillKeys = (): JSX.Element[] | null => {
+    if (pool.essentialSkills && pool.essentialSkills[0]) {
+      const chipsArray = pool?.essentialSkills?.map((skill) => (
+        <Chip
+          key={`essentialSkill-${skill.id}`}
+          mode="outline"
+          color="neutral"
+          label={getLocalizedName(skill.name, intl)}
+        />
+      ));
+      return chipsArray;
+    }
+    return null;
+  };
+
+  return (
+    <div
+      className="card"
+      data-h2-display="base(block)"
+      data-h2-shadow="base(s)"
+      data-h2-max-width="base(40rem)"
+    >
+      <div
+        className="card__body"
+        data-h2-background-color="base(dt-white)"
+        data-h2-padding="base(x.75)"
+      >
+        <h6
+          data-h2-font-weight="base(700)"
+          data-h2-padding="base(0, 0, x0.75, 0)"
+        >
```

Comment:
This may lead to some a11y issues related to the heading ranks. I'd suggest using the `Heading` component and drilling a level prop down.

```tsx
import Heading, { type HeadingLevel } from "@common/components/Heading";

...
interface PoolCardProps {
  ...
  headingLevel?: HeadingLevel;
}

// Note: You can set the default to w/e we will use most commonly (I think for the first use it is h3, though
const PoolCard = ({ pool, headingLevel = "h3" }: CardProps & React.HTMLProps<HTMLDivElement>) => {
  ...
    <Heading level={headingLevel} size="h6" data-h2-margin="base(0, 0, x.75, 0)">
      ...
    </Heading>
...
```

REF: https://www.w3.org/WAI/tutorials/page-structure/headings/#:~:text=Heading%20ranks,-Nest%20headings%20by&text=The%20most%20important%20heading%20has,of%20the%20higher%20ranked%20section.

Reply:
okay https://github.com/GCTC-NTGC/gc-digital-talent/pull/4233/commits/d0b44396a67bc8e7b787cb643374022bc7c68a15